healthcareai/tests/test_dev_supervised_model_class.py

```diff
@@ -24,75 +25,67 @@ def setUp(self):
         self.o.data_preparation(impute=True)
         self.o.random_forest(cores=1)
 
-    def runTest(self):
-
         self.assertAlmostEqual(np.round(self.o.au_roc, 6), 0.965070)
 
-    def tearDown(self):
-        del self.o
-
 
 class TestRFDevTuneTrueRegular(unittest.TestCase):
-    def setUp(self):
-        df = pd.read_csv(fixture('HCPyDiabetesClinical.csv'),
-                         na_values=['None'])
+    def test_random_forest_dev_tune_true_succeeds(self):
+        df = pd.read_csv(fixture('HCPyDiabetesClinical.csv'), na_values=['None'])
 
         # Drop uninformative columns
         df.drop(['PatientID', 'InTestWindowFLG'], axis=1, inplace=True)
 
         np.random.seed(42)
-        self.o = DevelopSupervisedModel(dataframe=df, model_type='classification',
-                                        predicted_column='ThirtyDayReadmitFLG')
-        self.o.data_preparation(impute=True)
+        o = DevelopSupervisedModel(dataframe=df, model_type='classification', predicted_column='ThirtyDayReadmitFLG')
+        o.data_preparation(impute=True)
```

Comment:
In this new framework, are you envisioning data_preparation as being the only user-specified method call that precedes the calling of the algorithms themselves? Did it save a lot of headache to surface o.data_preparation to the user (rather than impute being a *class* argument)?